rest/src/main/resources/flaxo.properties

```diff
@@ -1,2 +1,5 @@
 flaxo.github.hook.url=${REST_URL}/github/hook
 flaxo.travis.hook.url=${REST_URL}/travis/hook
+flaxo.plagiarism.analyser=gitplag
```

Comment:
Let's use some environment variable for this value as well. And I think that it can be helpful to use `moss` plagiarism analyser as a default one. Just because it is easier to use during the development.

Reply:
Done